backend/search/csv_service.py

```diff
@@ -0,0 +1,149 @@
+from django.http import HttpResponse
+from neomodel import db
+
+URI = "bolt://localhost:7687"
+AUTH = ("neo4j", "gradvek1")
```

Comment:
Shouldn't have URI and AUTH here, all the database login handled in startup.py and stored in .env or environment variables.

Should just be able to use the neomodel db functions as long as startup has happened.

Reply:
@kevinpatel24 - this merge includes your CSV branch could you take a look at @nsheely's comment

Reply:
Just made a push to the fe-file-uploads branch that removes those two lines. I tested without them and the path still works as Nathan said they would